pkg/controller/compliancesuite/compliancesuite_controller.go

```diff
@@ -254,7 +251,8 @@ func (r *ReconcileComplianceSuite) reconcileScanRemediations(suite *complianceop
 		resultRemediations, err := parseResultRemediations(r, scan.Name, scan.Namespace, &cm, logger)
 		if err != nil {
 			logger.Error(err, "cannot parse scan result")
-			return err
+			// If the results are not parseable, we cannot recover from this.
+			return common.WrapNonRetriableCtrlError(err)
```

Comment:
So earlier I said that in a future patch we might want to wrap errors with %w. This is one place where I think it would be nice so that the context is propagated up the stack.